### ISSUE TITLE
bastion_host looks for ~/.ssh/id_rsa with ssh_agent_auth = true

### ISSUE THREAD

**Issue by yvespp** [yamamoto-febc/terraform-provider-rke#42]
I'm using `bastion_host` with `ssh_agent_auth = true` but the provider still looks for `~/.ssh/id_rsa`. Log:
```
data.rke_node_parameter.nodes[2]: Refreshing state...
data.rke_node_parameter.nodes[1]: Refreshing state...
data.rke_node_parameter.nodes[0]: Refreshing state...
rke_cluster.cluster: Refreshing state... (ID: 10.22.141.17)

Error: Error refreshing state: 1 error(s) occurred:

* module.rancher_cluster.rke_cluster.cluster: 1 error(s) occurred:

* module.rancher_cluster.rke_cluster.cluster: rke_cluster.cluster: Error while reading SSH key file: open /home/yvespp/.ssh/id_rsa: no such file or directory
```

When I do a `touch ~/.ssh/id_rsa` it works, but in the next runs of terraform apply it has changes in the `bastion_host`:
```
Terraform will perform the following actions:
  ~ module.rancher_cluster.rke_cluster.cluster
      bastion_host.0.port:         "22" => "0"
      bastion_host.0.ssh_key_path: "~/.ssh/id_rsa" => ""
```

Config:
```
resource rke_cluster "cluster" {
  nodes_conf = ["${data.rke_node_parameter.nodes.*.json}"]
  ssh_agent_auth = true
  cluster_name = "${var.cluster_name}"
  kubernetes_version = "${var.kubernetes_version}"
  bastion_host = {
    address        = "${vsphere_virtual_machine.vm.*.default_ip_address[0]}"
    ssh_agent_auth = true
    user           = "rancher"
  }

  private_registries = {
    is_default = true
    password = "pw"
    url = "docker-registry.mycorp.com"
    user = "yvespp"
  }
  services_kube_api = {
    service_cluster_ip_range = "172.23.0.0/16"
  }
  services_kube_controller = {
    cluster_cidr = "172.28.0.0/14"
    service_cluster_ip_range = "172.23.0.0/16"
  }
  services_kubelet = {
    cluster_dns_server = "172.23.0.10"
  }
}
```

**Comment by mcapuccini** [comment]
I may be naive, but did you set a password in your config?

**Comment by yvespp** [comment]
You mean for the ssh Key? The key is already loaded in my ssh agent (Pagent of putty) so no password is needed.

**Comment by yamamoto-febc** [comment]
Hi @yvespp,

Thank you for this reporting!

First, looking for `~/.ssh/id_rsa` seems to be the default behavior of RKE.

I tried `rke up` with following config:

```yaml
nodes:
- address: host1.example.com
  role:
  - controlplane
  - worker
  - etcd
  user: rancher
  ssh_key_path: /home/user/.ssh/bastion_rsa
bastion_host:
  address: bastion.example.com
  user: example
  ssh_agent_auth: true
```

But I got following errors:

```
$ rke --version

rke version v0.1.13

$ rke up

INFO[0000] Building Kubernetes cluster                  
FATA[0000] Error while reading SSH key file: open /Users/foobar/.ssh/id_rsa: no such file or directory 

```

So I think this is RKE's issue.

Next,  

>When I do a touch ~/.ssh/id_rsa it works, but in the next runs of terraform apply it has changes in the bastion_host:

This is rke-provider's issue. 
These are setting `Computed: false` on their [schema define](https://github.com/yamamoto-febc/terraform-provider-rke/blob/0.6.1/rke/resource_rke_cluster.go#L567-L599).

I think they should be set `Computed: true`. So I'm going to update their schema define.

**Comment by yamamoto-febc** [comment]
We have updated schema define and released [v0.6.2](https://github.com/yamamoto-febc/terraform-provider-rke/releases/tag/0.6.2).
Please try it :pray:

**Comment by yvespp** [comment]
@yamamoto-febc I tested it and it works! Thanks for fixing and releasing it so quickly!
